node/pkg/adminrpc/adminserver.go

```diff
@@ -1162,7 +1162,18 @@ func (s *nodePrivilegedService) SignExistingVAA(ctx context.Context, req *nodev1
 	}
 
 	// Add local signature
-	newVAA.AddSignature(s.gk, uint8(localGuardianIndex))
+	sig, err := s.guardianSigner.Sign(v.SigningDigest().Bytes())
+	if err != nil {
+		panic(err)
+	}
+
+	sigData := [65]byte{}
```

Comment:
As @bruce-riley can likely attest, I do hate [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)). I personally know that an ecdsa signature is 64 bytes, but these magic numbers are best (in the future, or a followup PR) as a constant. There are often bugs when magic numbers aren't updated in the future that can be mitigated by using constants when sensible.